e2e/specs/st_experimental_get_query_params.spec.js

```diff
@@ -27,14 +27,20 @@ describe("st.experimental_get_query_string", () => {
   it("shows query string correctly", () => {
     cy.get(".element-container [data-testid='stMarkdownContainer']").should(
       "have.length",
-      1
+      2
     );
-    cy.contains(
+    cy.getIndexed(".element-container [data-testid='stMarkdownContainer']", 0).contains(
+      "Please replace st.experimental_get_query_params with st.query_params. " +
+      "st.experimental_get_query_params will be removed after 2024-04-13. " +
+      "Please refer to our docs page to replace experimental_get_query_params."
```

Comment:
Worth running by product for the exact wording, but I feel like this last line is awkward.

It'd feel less redundant to say something like "[Please] refer to ours docs page for more information" given that we've already mentioned `experimental_get_query_params` twice in the two lines directly above this one. (May also make sense to drop the second please for the same reason of the wording feeling pretty redundant).